avaframe/com1DFAPy/com1DFA.py

```diff
@@ -1269,7 +1269,16 @@ def prepareArea(releaseLine, dem, relThList=''):
     if relThList != '':
         Raster = np.zeros(np.shape(dem['rasterData']))
         for rast in RasterList:
-            Raster = Raster + rast
+            ind1 = Raster > 0
+            ind2 = rast > 0
+            indMatch = np.logical_and(ind1, ind2)
+            try:
+                message = 'Release area features are overlaping - this is not allowed'
+                assert indMatch.any() == False, message
+                Raster = Raster + rast
+            except AssertionError:
+                log.error('Release area features are overlaping - this is not allowed')
```

Comment:
This is redundant, why not use the message you provide with the assert check?

Reply:
done